src/spans/redisSpan.js

```diff
@@ -0,0 +1,25 @@
+import { getBasicChildSpan, REDIS_SPAN } from './awsSpan';
+import { payloadStringify } from '../utils/payloadStringify';
+
+export const createRedisSpan = (spanId, requestMetadata, redisFields) => {
+  const baseSpan = getBasicChildSpan(spanId, REDIS_SPAN);
+  return {
+    ...baseSpan,
+    started: requestMetadata.started,
+    requestCommand: redisFields.command.command,
```

Comment:
Is `redisFields.command` never `undefined`?

Reply:
Fixed